sorc/chgres_cube.fd/program_setup.F90

```diff
@@ -94,6 +94,8 @@ module program_setup
  logical, public                 :: convert_atm = .false. !< Convert atmospheric data when true.
  logical, public                 :: convert_nst = .false. !< Convert nst data when true.
  logical, public                 :: convert_sfc = .false. !< Convert sfc data when true.
+ integer, public                 :: wam_start_date        !< wam start date.
```

Comment:
There are already namelist variables for cycle month, day and hour.  We can add a new variable for year if you need it.   Otherwise 'wam_start_date' is a bit redundant.

Reply:
Agree with you.  I will remove wam_start_date and add cycle_year for namelist etc.